Sources/Plasma/FeatureLib/pfMoviePlayer/plMoviePlayer.cpp

```diff
@@ -327,13 +328,21 @@ void plMoviePlayer::IProcessVideoFrame(const std::vector<blkbuf_t>& frames)
     }
 
     if (img) {
+        if (fLastImg) {
+            vpx_img_free(fLastImg);
+        }
+        fLastImg = img;
+    }
+    
+    // We need to refill the buffer if there is a new image or a new buffer
+    if (img || !fTexture->GetDeviceRef()) {
```

Comment:
Should this use `fLastImg` instead of `img`? In theory it shouldn't matter?

Reply:
It needs to use img to know if a new frame was actually loaded. fLastImg would be populated either way.